src/main/java/com/cleanroommc/groovyscript/compat/mods/astralsorcery/Lightwell.java

```diff
@@ -177,6 +179,7 @@ public boolean validate() {
             }
             if (this.output == null) out.add("No output specified.").error();
             if (this.catalyst == null) out.add("No catalyst specified.").error();
+            if (IngredientHelper.overMaxSize(this.catalyst, 1)) out.add("Catalyst must have a stack size of 11.").error();
```

Comment:
11? additionally, should add something like `validateCustom` to AbstractRecipeBuilder - perhaps `validateSize(int size, String type)`

Reply:
typo

Reply:
Actually this class doesn't inherit AbstractRecipeBuilder 😔